src/Collection.php

```diff
@@ -553,10 +553,6 @@ public function map(callable ...$callbacks): CollectionInterface
 
     public function match(callable $callback, ?callable $matcher = null): CollectionInterface
     {
-        // @todo: Rename this in next major version.
-        // We cannot use Match::class because PHP 8 has
-        // a new "match" function and we cannot use the same name.
-        // @See https://github.com/loophp/collection/issues/56
```

Comment:
I don't think this `@todo` was needed here anymore, right? This method works in php 8

Reply:
Indeed, it's a leftover.